DeviceSettings/res/values/aosip_strings.xml

```diff
@@ -17,7 +17,7 @@
     <string name="display_title">Display</string>
     <string name="srgb_mode_summary"></string>
     <string name="srgb_mode_title">sRGB</string>
-    <string name="hbm_mode_summary"></string>
+    <string name="hbm_mode_summary">Increases the maximum brightness of the display to improve visibility under sunlight. Increases power consumption.</string>
```

Comment:
```suggestion
    <string name="hbm_mode_summary">Light the display in a brighter than the normally allowed maximum brightness. Using for a long period of time may harm the display</string>
```

As power consumption is pretty clear and it doesn't increases the max, it enforces it.

Reply:
maybe "damage" is better